java_console/io/src/main/java/com/rusefi/SimulatorExecHelper.java

```diff
@@ -14,7 +14,7 @@ public class SimulatorExecHelper {
     private final static NamedThreadFactory THREAD_FACTORY = new NamedThreadFactory("SimulatorExecHelper", true);
 
     private static final String SIMULATOR_BINARY_NAME = "fome_simulator";
-    private static final String SIMULATOR_BINARY_PATH = "../simulator/build";
+    private static final String SIMULATOR_BINARY_PATH = "../simulator/build/";
```

Comment:
@nmschulte I am really sorry but where do you see "simulator" directory/folder in current snapshot bundle?

Reply:
It's not in the snapshot bundle; the console uses `SimulatorHelper`, which dishes out to `SimulatorExecHelper` to wrangle the actual executable (and its output); it has its own path specification: https://github.com/FOME-Tech/fome-fw/blob/master/java_console/ui/src/main/java/com/rusefi/SimulatorHelper.java#L17

`SimulatorExecHelper` is used by `SimulatorFunctionalTestLauncher`/`IoUtil` to run functional tests against the simulator (but the file notes it may not work correctly anymore).  The change here on this line probably also broke these `SimulatorFunctionalTestLauncher` integration, because `"../simulator/build"` + `"fome_simulator"` misses the path separator (`"/"`), so fixing that here (and also no longer concatenating these strings).